USE_UV_LAMP: true
UV_LAMP_WARMUP_TIME: 10.0
USE_LED_AS_UV: false
USE_STOP_AS_UV: false
MISSION_DIR: "/root/.ais/missions/"
MISSION_NAME: "example_mission.csv"
MAX_RAIL_LENGTH: 107
RETURN_TO_RAIL_HOME: true
RETURN_TO_RAIL_HOME_WITH_UV_ON: true
RETURN_TO_RAIL_HOME_SPEED: 0.78
RETURN_HOME_INTENSITY: [100, 100,  100,  100,  100,  100,  100,  100]
RAIL_TRANSITION_INTENSITY: [25, 25,  25,  25,  25,  25,  25,  25]
FORWARD_DIRECTION_ON_RAIL: true
ROBOT_GOAL_STOP_TIME: 0
MAX_RAIL_SPEED: 0.78
RAIL_IS_LEFT_ADJACENT: true
DEMO_MULTIPLE_RAILS: false
#OFF_RAIL_DETECTION must be true for either AUTONOMOUS to be enabled
OFF_RAIL_DETECTION: true
OFF_RAIL_DETECTION_AUTONOMOUS: false
OFF_RAIL_DETECTION_SEMI_AUTONOMOUS: false
IRRADIANCE_SPEED_CONTROL: false
RAIL_MISSION_COUNTER: 0
NUMBER_OF_RAIL_MISSIONS: 1
LOW_BATTERY: 35
CRITICAL_BATTERY: 25
ESTIMATED_BATTERY_USAGE: 0
MINIMUM_BATTERY: 80
DIAGNOSTIC_SAFETY: false
STRESS_TEST: false
